### ISSUE TITLE
Institution without role in academic course

### ISSUE THREAD

**Issue by natuk** [Worlding-Public-Cultures/wpc-website#6]
At the moment the academic course has a field directly linking to Institution. I do not think this is correct. Shouldn't the Institution link through actor role? Or are we expecting all Institution involved with a course to be the academic institutions that the course is run? Also, shouldn't the Institution be linked to the academic programme instead of the course?

**Comment by mbelhu** [comment]
You're right. The institution should be linked to the academic programme. 
In any case, to avoid duplications, would it be better to remove it and provide the institution information through actors or even location fields?
